src/plugins/hotkeyPlugin.ts

```diff
@@ -311,15 +313,21 @@ export const eventToCombination = (event: KeyboardEvent): HotkeyCombination => {
   if (event.metaKey) {
     recordedCombination += "Meta ";
   }
-  if (event.key === " ") {
-    recordedCombination += "Space";
-  } else {
-    if (["Control", "Shift", "Alt", "Meta"].includes(event.key)) {
-      recordedCombination = recordedCombination.slice(0, -1);
-    } else {
-      recordedCombination +=
-        event.key.length > 1 ? event.key : event.key.toUpperCase();
-    }
+  // event.codeから以前のevent.key形式へと変換
+  // 列挙するのは将来的に変更する
+  const eventKey = event.code
+    .replace(/Key|Digit|Numpad/, "")
+    .replace("Minus", "-")
+    .replace("Slash", "/");
+  // 英字 数字 上下左右 Enter Space Backspace Delete Escape F1~ - /のみ認める
+  if (
+    /^([A-Z]|\d|Arrow(Up|Down|Left|Right)|Enter|Space|Backspace|Delete|Escape|F\d+|-|\/)$/.test(
+      eventKey,
+    )
+  ) {
+    recordedCombination += eventKey;
   }
+  // 末尾がスペースならそれを削除
+  recordedCombination = recordedCombination.replace(/\s$/, "");
```

Comment:
これってなんででしたっけ 👀 
全方向Arrowなどを登録できる実装の名残り･･･？だとしたら一旦消せそう･･･？

Reply:
補助キーでスペースがついてしまうのでそれを消しています（例えばシフトだけを押したら"shift "となるので最後のスペースを削除してます）

Reply:
あーなるほどです！！
（そもそもスペース区切りじゃなくてArrayだったほうが扱いやすい気がしますね･･･ 😇 ）

ちょっとコメントだけ変更させてください！（意図をコメントに書く）
```suggestion
  // 修飾キーのみだった場合末尾がスペースになるので削除
  recordedCombination = recordedCombination.replace(/\s$/, "");
```